# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1276361703 14400
#      Sat Jun 12 12:55:03 2010 -0400
# Node ID a3c04f036352dbd7352c2e0bc2ec9d7f4bf9971d
# Parent  da51afafca807a5696e94fdfd45a3d1da50dedab
gammainc.cc: Fix doc-string typo.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-06-12  Ben Abbott <bpabbott@mac.com>
+
+	* DLD-FUNCTIONS/gammainc.cc: Fix doc-string typo.
+
 2010-06-11  David Bateman  <dbateman@free.fr>
 
 	* DLD-FUNCTIONS/gammainc.cc: Implement complementary incomplete
 	gamma function.
 
 2010-06-10  Ben Abbott <bpabbott@mac.com>
 
 	* data.cc: Fix test for concatentating empty nd-arrays.
diff --git a/src/DLD-FUNCTIONS/gammainc.cc b/src/DLD-FUNCTIONS/gammainc.cc
--- a/src/DLD-FUNCTIONS/gammainc.cc
+++ b/src/DLD-FUNCTIONS/gammainc.cc
@@ -64,17 +64,17 @@ for each element of @var{x} and vice ver
 If neither @var{x} nor @var{a} is scalar, the sizes of @var{x} and\n\
 @var{a} must agree, and @var{gammainc} is applied element-by-element.\n\
 \n\
 By default the incomplete gamma function integrated from 0 to @var{x} is\n\
 computed. If \"upper\" is given then the complementary function integrated\n\
 for @var{x} to infinity is calculated. It should be noted that\n\
 \n\
 @example\n\
-gammainc (@var{x}, @var{a}) = 1 - gammainc (@var{x}, @vaar{a}, \"upper\")\n\
+gammainc (@var{x}, @var{a}) = 1 - gammainc (@var{x}, @var{a}, \"upper\")\n\
 @end example\n\
 @seealso{gamma, lgamma}\n\
 @end deftypefn")
 {
   octave_value retval;
   bool lower = true;
 
   int nargin = args.length ();
